.github/workflows/codeql-analysis.yml

```diff
@@ -59,9 +59,16 @@ jobs:
     #    and modify them (or add more) to build your code if your project
     #    uses a compiled language
 
-    #- run: |
-    #   make bootstrap
-    #   make release
-
+    - name: Set up java
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    
+    - name: Build with Gradle
+      run: ./gradlew build
+        
```

Comment:
Might be nice to remove all the commented out bits from the default file and leave a comment explaining why we had to build manually instead of using the autobuild step